src/include/sof/audio/module_adapter/iadk/api_version.h

```diff
@@ -19,4 +19,10 @@
 #define IADK_MODULE_API_CURRENT_VERSION	MODULE_API_VERSION_ENCODE(IADK_MODULE_API_MAJOR_VERSION, \
 	IADK_MODULE_API_MIDDLE_VERSION, IADK_MODULE_API_MINOR_VERSION)
 
+/* Defines the size of the space reserved within ModuleHandle for SOF to store its private data.
+ * This size comes from the IADK header files and must match the IADK API version.
+ * Please do not modify this value!
+ */
+#define IADK_MODULE_PASS_BUFFER_SIZE		320
```

Comment:
it's better now because it's an IADK-specific header, but I still don't think it should be in an API-version header. Just look at what this header is defining. A buffer size really doesn't fit here very well

Reply:
@lyakh This buffer is reserved on the library side. Its size has changed depending on the API version and that's why I think it's good to have these things together.

Reply:
I'm not working with IADK so my opinion isn't very important here, but it just looks weird. That header seems to define macros to *work* with the API version - to pack and unpack it, maybe defining some version values etc. It isn't defining version-dependent things. But it's up to you of course.